client/analytics/components/summary/index.js

```diff
@@ -0,0 +1,108 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { map } from 'lodash';
+import { withSelect } from '@wordpress/data';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { formatCurrency } from 'lib/currency';
+import { getNewPath } from 'lib/nav-utils';
+import { SummaryList, SummaryListPlaceholder, SummaryNumber } from '@woocommerce/components';
+import { getCurrentDates, getDateParamsFromQuery } from 'lib/date';
+import { getSummaryNumbers } from 'store/reports/utils';
+class ReportSummary extends Component {
```

Comment:
:+1: `ReportSummary`. I think the filename should also reflect this, `/report-summary/`